### DISCUSSION TITLE
Turn on 기능과 Shutdown기능 구현 질문

### DISCUSSION THREAD

**Discussion by Beantree09** [KimMin-Gwan/ICT-convergence-project-contest-2023#35] [Q&A] [answered]
리모컨의 shutdown과 turn on 기능은 하나의 주파수로 이루어져 있는데 구분을 해야 하는 이유가 있나요?

**Comment by KimMin-Gwan** [comment, marked as answer]
구분하지 않아도 된다면 구분하지 않아도 됩니다.
가이드라인을 수정 하려면 수정된 가이드 라인을 제시해주시면 됩니다.
